website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx

```diff
@@ -102,27 +102,27 @@ The way you set up and configure a new policy set depends on your workflow and w
-   - **(OPA Only) Overrides**:  Choose whether users with override policy permissions can let Terraform apply plans that have [mandatory policy](#policy-enforcement-levels) failures.
-   - **(VCS Only) VCS Branch**: Specify the branch within your VCS repository where Terraform Cloud should import new versions of policies. If you do not set this field, Terraform Cloud uses the default branch of the VCS repository you selected.
-   - **(VCS Only) Policies Path**: Specify the sub-directory in your VCS repository containing the policy set files. This action lets you maintain multiple policy sets within a single repository. Set this field to the directory path that contains the `sentinel.hcl` (Sentinel) or `policies.hcl` (OPA) configuration file for the policy set. You can start the path with `/`, but this is optional. Terraform Cloud assumes that relative paths originate from the root of the repository. If you do not set this field, Terraform Cloud uses the repository root directory.
-   - **(Managed Policies Only) Policies**: Select managed policies to add to the policy set. You can only add policies written with the policy framework you selected for the policy set.
+1. Choose a **Policy framework** for the policies you want to add. You can only create a policy set from policies written using the same framework. You cannot change the framework type after creation.
+1. Choose a policy set scope:
+   - **Policies enforced globally:** Terraform Cloud will automatically enforce this global policy set on all existing and future workspaces in the organization.
+   - **Policies enforced on selected projects and workspaces:** Use the text fields to search for and select workspaces and/or projects to enforce this policy set on. This affects all current and future workspaces for any selected projects.
+1. **(OPA Only)** Allow **Overrides** where users with override policy permissions can let Terraform apply plans that have [mandatory policy](#policy-enforcement-levels) failures.
+1. **(VCS Only)** Specify the **VCS branch** within your VCS repository where Terraform Cloud should import new versions of policies. If you do not set this field, Terraform Cloud uses the default branch of the VCS repository you selected.
+1. **(VCS Only)** Specify the **Policies path** in your VCS repository containing the policy set files. This action lets you maintain multiple policy sets within a single repository. Set this field to the directory path that contains the `sentinel.hcl` (Sentinel) or `policies.hcl` (OPA) configuration file for the policy set. You can start the path with `/`, but this is optional. Terraform Cloud assumes that relative paths originate from the root of the repository. If you do not set this field, Terraform Cloud uses the repository root directory.
```

Comment:
Suggesting shorteding this for brevity and readability:
```suggestion
1. **(VCS Only)** Specify where your policy set files live using the **Policies path**. This lets you maintain multiple policy sets within a single repository. Use a relative path from your root directory to the directory that contains either the `sentinel.hcl` (Sentinel) or `policies.hcl` (OPA) configuration files. If you do not set this field, Terraform Cloud uses the repository's root directory.
```

Reply:
Love it!